### ISSUE TITLE
Default value for works/editions ?

### ISSUE THREAD

**Issue by hsarrazin** [vlsergey/WE-Framework#22]
Hi, when used on wikisource project, the language (P407) is nearly always the main language of the project (i.e. en frws -> fr ; sometimes fro or frc, or other regional or historic variants).

Do you think it would be possible to automatically have the default language value for the wikisource projects, so that for most texts it would not be necessary to specifically edit it ?

Thanks

**Comment by vlsergeyatsbt** [comment]
There is 2 points:
* will be too simple to "forget" to erase default value (if required)
* there is no direct link between local language and language item in wikipedia (only backlink via P424 )

There will be small improvment in future version with last recently used values for entity fields. Thus one will need only 2 clicks to set recently used value (i.e. language). May be it will be sufficient for this task.

**Comment by vlsergey** [comment]
There are 2 points:

* will be too simple to "forget" to erase default value (if required)
* there is no direct link between local language and language item in wikipedia (only backlink via P424 )

There is be small improvment in current version with last recently used values for entity fields. Thus one will need only 2 clicks to set recently used value (i.e. language). May be it will be sufficient for this task.

**Comment by vlsergey** [comment]
@hsarrazin did you check the current scheme with latest used values?

**Comment by hsarrazin** [comment]
yup, but you need to click on it... for 95% of items, it would be the same language as the project...
